README.markdown

```diff
@@ -73,6 +73,17 @@ An array of services to start, normally `puppet` and `mcollective`. If the array
 
 Supports only RPM-based distros: Redhat and Centos 5/6/7.
 
+###Known Issues
+
+The puppet_agent module has several known issues on Windows. This module supports Windows Server 2003 or later.
+
+* For Windows Server 2003, only x86 is supported.
+* Because the module directory has changed, modules installed manually on individual agents cannot be found after upgrading to Puppet 4.x. You should reinstall these modules with `puppet module install`.
```

Comment:
@MikaelSmith I'll move this out as a separate known issue. The only reason it's called out as Windows is because it was given to me in a list of "Known Issues with Windows". Thank you for clearing it up!